librz/core/clang.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_types.h>
+#include <rz_core.h>
+#include <rz_lang.h>
+
+RZ_API RzCmdStatus rz_core_lang_plugins_print(RzLang *lang, RzCmdStateOutput *state) {
+	RzListIter *iter;
+	RzLangPlugin *h;
+	PJ *pj = state->d.pj;
+	rz_cmd_state_output_array_start(state);
+	if (!lang) {
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_list_foreach (lang->langs, iter, h) {
+		const char *license = h->license
+			? h->license
+			: "unknown";
+		switch (state->mode) {
+		case RZ_OUTPUT_MODE_JSON: {
+			pj_o(pj);
+			pj_ks(pj, "Name", h->name);
+			pj_ks(pj, "License", license);
+			pj_ks(pj, "Description", h->desc);
+			pj_end(pj);
+			break;
+		}
+		case RZ_OUTPUT_MODE_STANDARD: {
+			lang->cb_printf("%s: (%s) %s\n",
```

Comment:
just use `rz_cons_printf`. 

`lang->cb_printf` was necessary because this code was in librz/lang, but here you can directly call `rz_cons_printf`.